src/components/levels.js

```diff
@@ -53,16 +53,16 @@ This CSS will turn all Zebras a deep shade of purple!`,
     title: "Class",
     instructions: <p>It looks like this part of your safari has had a fire recently and your zebras have nothing to eat. Turn the safari green <Hex hexcode="#306230"/> again to make sure your animals are well fed.</p>,
     gamepieces: [
-        new stickers.Boabab('Bob', 10, 30, "#1d0b08", "plant"),
-        new stickers.Boabab('Bill', 65, 75, "#1d0b08", "plant"),
-        new stickers.Acacia('Ace', 10, 5, "#1d0b08", "plant"),
-        new stickers.Acacia('Aiden', 15, 50, "#1d0b08", "plant"),
-        new stickers.Acacia('Alicia', 30, 40, "#1d0b08", "plant"),
-        new stickers.ElephantGrass('Edward', 78, 6, "#1d0b08", "plant"),
-        new stickers.ElephantGrass('Elizabeth', 50, 12, "#1d0b08", "plant"),
-        new stickers.ElephantGrass('Ella', 65, 40, "#1d0b08", "plant"),
-        new stickers.ElephantGrass('Eduardo', 10, 75, "#1d0b08", "plant"),
-        new stickers.Zebra('zaira', 50, 50, "#fffcf5")
+        new stickers.Boabab('Bob', 10, 30, "#1d0b08", "boabab", "plant"),
+        new stickers.Boabab('Bill', 65, 75, "#1d0b08", "boabab", "plant"),
+        new stickers.Acacia('Ace', 10, 5, "#1d0b08", "acadia", "plant"),
```

Comment:
```suggestion
        new stickers.Acacia('Ace', 10, 5, "#1d0b08", "acacia", "plant"),
```

Reply:
we probably don't want to be pass in the component's name to be a class variable. This is duplication of code, and is prone to human error (see case study 1, this one that kyle pointed out :0). I look at this is think that there's probably a way to get an object's classname ;)

lmk if you need more help, but overall, we just shouldn't have the `component_name` variable overall